src/services/db.py

```diff
@@ -1,16 +0,0 @@
-from remodel.models import Model
```

Comment:
Agreed; this file is currently needed to start the DB. Let @sungbae tackle this.